static/app/components/events/highlights/util.tsx

```diff
@@ -170,3 +171,35 @@ export function getHighlightTagData({
     originalTag: tagMap[tagKey]?.tag ?? {key: tagKey, value: EMPTY_HIGHLIGHT_DEFAULT},
   }));
 }
+
+/**
+ * Returns a label that can be used in the Event Highlight Summary.
+ * Currently only used for JavaScript-based events.
+ */
+export function getRuntimeLabelAndTooltip(
+  event: Event,
+  knownRuntimeType?: {isBackend?: boolean; isFrontend?: boolean}
+): {label: string; tooltip: string} | null {
+  if (!event.sdk?.name.includes('javascript')) {
+    return null;
+  }
+
+  if (knownRuntimeType?.isBackend) {
+    return {label: t('Backend'), tooltip: t('Error from Server, Edge or Worker Runtime')};
+  }
+  if (knownRuntimeType?.isFrontend) {
+    return {label: t('Frontend'), tooltip: t('Error from Client (inside Browser)')};
```

Comment:
```suggestion
    return {label: t('Frontend'), tooltip: t('Error from Client (e.g. Browser)')};
```

maybe like this?

Reply:
Yeah, I wasn't sure if I can/should use abbreviations 🤔